### ISSUE TITLE
newer compression module

### ISSUE THREAD

**Issue by Enalmada** [Sly777/ran#543]
Upgrade from compression to [shrink-ray-current](https://www.npmjs.com/package/shrink-ray-current).  This is a drop in replacement.

At the default level, Akamai found that brotli averages [20% faster and 10% smaller](https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html) than older compression methods.     I just put it into the RAN example to test it out and it seems to work great.

"The combination of caching and use of better compression algorithms makes shrink-ray serve static files in our benchmark 3x faster than compression while using only one quarter as much CPU time." ~ https://www.npmjs.com/package/shrink-ray-current

**Comment by Sly777** [comment]
Hey @Enalmada, I'm not sure about this for now because AFAIK nodejs will support this internal mode in the future so maybe we should wait that. WDYT?

**Comment by Enalmada** [comment]
Supporting the latest compression schemes is a perfect complement to the  "performance first" claim on the RAN homepage.  I think the only reason not to use shrink-ray-current is if you were aware of a better module that provided modern encodings.  I think you need to have a really compelling case to not default new users to the encoding used by the biggest sites in the world: 

![image](https://user-images.githubusercontent.com/1892132/45375783-d8baf900-b5aa-11e8-911f-e142865ab7d6.png)

![image](https://user-images.githubusercontent.com/1892132/45377176-aa3f1d00-b5ae-11e8-8a3d-c648ca0688d3.png)

**Comment by Enalmada** [comment]
After researching about best practices of compression with node, I have changed my mind a bit on this.  While I believe that Akamai has proven that brotli at default levels is better for the web than gzip, there is evidence that node should not be the process doing any compression.  See point 3:
https://goldbergyoni.com/checklist-best-practice-of-node-js-in-production/

TL;DR a real proxy should be doing the compression due to node threading model.   I believe it could be worth commenting out compression in the server.js with comments stating this should only be enabled if there is no better way to achieve it.  (AWS Cloudfront, Beanstalk and Lambda will compress for you)
